app/assets/stylesheets/application/organisms/_item_list.scss

```diff
@@ -0,0 +1,111 @@
+.item-list {
+  background-color: $pale-gray-color;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets